.Dislocation Motion.tmp.nlogo

```diff
@@ -275,104 +283,113 @@ to calculate-fl-positions
   ]
 end
 
-to identify-force-atoms
+to identify-force-atoms ; (find the atoms closest to the force line that will be the ones receiving the external force)
   (ifelse force-mode = "Shear" [
-    ask atoms [ set ex-force-applied?  0 ]
-    ask min-n-of num-forced-atoms atoms with [ ycor >= median-ycor ] [ distancexy upper-left-fl ycor] [
-      set ex-force-applied?  1
+    ask atoms [ set ex-force-applied?  False ]
+    let forced-atoms atoms with [ ycor >= median-ycor and (distancexy upper-left-fl ycor) <= 1]
+    set num-forced-atoms count forced-atoms
+    ask forced-atoms [
+      set ex-force-applied?  True
     ]
     ]
     force-mode = "Compression" [
-      ask atoms [ set ex-force-applied?  0 ]
-      ask min-n-of num-forced-atoms atoms [ distancexy left-fl ycor] [
-        set ex-force-applied?  1
+      ask atoms [ set ex-force-applied?  False ]
+      let forced-atoms atoms with [ (distancexy left-fl ycor) <= 1]
+      set num-forced-atoms count forced-atoms
+      ask forced-atoms [
+        set ex-force-applied?  True
     ]
   ]) ; for tension, the same atoms in the left shoulder of the sample always receive the force
 end
 
 
 to check-eq-adj-force ; (check equilibrium, adjust force)
-  if f-app != f-app-prev [ set f-app-auto f-app ] ; updates f-app-auto if the user manually changed f-app
-  if precision prev-length 4 = precision (right-fl - left-fl) 4 [ set f-app-auto f-app-auto + .1 ] ; increments f-app-auto if the sample has reached an equilibrium
-  set prev-length (right-fl - left-fl)
-  set f-app-prev f-app
+  if precision prev-length 6 >= precision (right-edge - left-fl) 6 [ set f-app precision (f-app + .01) 3 ]
```

Comment:
Let's try incrementing by a smaller amount than .01. I don't think it will make the model much slower since it will get increment more often, and I think it will make the stress-strain curve smoother.

Reply:
![image](https://user-images.githubusercontent.com/44755643/92153076-b5ba2d00-edf1-11ea-8c47-a44771655620.png)
it is perhaps worse to do this

Reply:
![image](https://user-images.githubusercontent.com/44755643/92153167-d2eefb80-edf1-11ea-81aa-88e873ebf60f.png)
Depends on your definition of worse I suppose. The overall shape is smoother in the first image, but there's much more zigzagging

Reply:
![image](https://user-images.githubusercontent.com/44755643/92154626-1480a600-edf4-11ea-8eb2-adb77314f8d4.png)
happy medium? or the worst of both worlds?